index.js

```diff
@@ -72,7 +72,7 @@ function jaegerPlugin (fastify, opts, next) {
 
   function onResponse (req, reply, done) {
     const span = tracerMap.get(req)
-    span.setTag(Tags.HTTP_STATUS_CODE, reply.res.statusCode)
+    span.setTag(Tags.HTTP_STATUS_CODE, reply.statusCode)
```

Comment:
According to the deprecation message the change should be:
```suggestion
    span.setTag(Tags.HTTP_STATUS_CODE, reply.raw.statusCode)
```